symengine/sets.h

```diff
@@ -17,6 +17,7 @@ class Set : public Basic
 public:
     virtual bool is_Interval() const = 0;
     virtual bool is_EmptySet() const = 0;
+    virtual bool is_UniversalSet() const = 0;
```

Comment:
Do we need this to be a pure virtual function? This can be easily achieved by checking `is_a<UniversalSet>(*this)`